src/SymbolicUtils.jl

```diff
@@ -35,7 +35,7 @@ include("rewriters.jl")
 using .Rewriters
 
 using Combinatorics: permutations, combinations
-export @rule, @acrule, RuleSet
+export @rule, @acrule, RuleSet, @capture
```

Comment:
One thing to think about is if there are any packages that do
```julia
using MacroTools, SymbolicUtils
```
If so, this addition will be a breaking change for them. Not sure that's really worth the trouble of not adding this functionality, but I thought I'd bring it up.

Reply:
We could decide not to export, or call it `@match` instead perhaps?

Reply:
`@match` also exists elsewhere so it doesn’t really help. I think it’s probably fine to export, but we should bump the version.